governance.md

```diff
@@ -0,0 +1,147 @@
+
+The key responsibiliy of the Core Team is to define and hold the vision and values of the NUnit Community and ensure that we stick to them. They should periodically review and if necessary update our vision and values documents and should ensure that all committers are aware of them.
+
+The Core Team recruits, selects and appoints project leaders and confirms the appointment of committers on all projects. It also sets standards that are needed in order for our projects to interoperate with one another. This may include such things as coding standards, use of GitHub and inter-project decision making. It should endeavor to avoid micro-managing the software projects and should leave them as much freedom as is reasonable.
+
+The Core Team will conduct a regular review of each software project, annually for core projects and every two years for others. Additional review may be called for if a project is having problems. The review should look at releases, bug fixing and other indications that the project is performing as desired.
+
+The Core Team is called upon to resolve conflicts between software projects or if a consensus cannot be reached on an important decision within a single project. Individual projects may also refer important decisions to the Core Team, particularly those that reflect our community values. Issues of copyright and licensing are always referred to the core team.
+
+The Core Team is responsible for approving, maintaining and amending these governance policies as needed.
+
+Membership in the Core Team is by invitation of the existing members. A nomination will result in discussion and possibly a vote by the existing members. As with all its decisions, the Core Team will attempt to reach a consensus on new members. If a vote is required, the new member must be approved by 2/3 of the membership.
+
+Core Team members remain on the team until they choose to retire or until 2/3 of the members vote to remove them.
+
+The initial composition of the Core Team is
+* Charlie Poole
+* Rob Prouse
+* Chris Maddock
+* Terje Sandstrom
+* Jamie Cansdale
+
+### Core Team Lead
+
+The Core Team Lead is a single individual, voted for by the Core Team members. Once someone has been appointed Lead, they remain in that role until they choose to retire, or the Core Team casts a two-thirds majority vote to remove them.
+
+The Lead has no additional authority over other members of the Core Team: the role is that of a coordinator and facilitator. The Chair is also expected to ensure that all governance processes are adhered to, and has the casting vote when the project fails to reach consensus.
+
+>**Reviewers:** If you like Organization Lead better, that's OK with me too. I considered Community Leader as well, but it seemed to overstate the case. The role is more like the go-between from the Core Team to the Community.
+
```

Comment:
I don't have a strong opinion, I only lean towards Organization Lead because it maps to the GitHub organization that encompasses all of the projects/repos within the organization.